src/integration-test/kotlin/integration/btc/BtcPkPreGenIntegrationTest.kt

```diff
@@ -0,0 +1,59 @@
+package integration.btc
+
+import com.github.kittinunf.result.failure
+import config.loadConfigs
+import integration.helper.IntegrationHelperUtil
+import kotlinx.coroutines.experimental.async
+import org.bitcoinj.wallet.Wallet
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.fail
+import provider.TriggerProvider
+import provider.btc.BtcKeyGenSessionProvider
+import registration.btc.key.BtcPkPreGenConfig
+import registration.btc.key.main
+import sidechain.iroha.IrohaInitialization
+import util.getRandomString
+import java.io.File
+
+class BtcPkPreGenIntegrationTest {
+    init {
+        IrohaInitialization.loadIrohaLibrary()
+            .failure { ex ->
+                println(ex)
+                System.exit(1)
```

Comment:
I think you can use `fail { ex }`

Reply:
yes we can. but this is a common pattern in d3 to load Iroha like that.

Reply:
But this is a test

Reply:
Ok. I will change it. But this may affect plenty of files, because I will change it in every test to keep it in one style.